src/main/javacc/se/jsannemo/spooky/compiler/Parser.jj

```diff
@@ -120,11 +121,16 @@ boolean BoolLiteral() : {
 // Program file
 Program Start() : {
     Program.Builder program = Program.builder();
+    VarDecl global;
     Function func;
     FunctionDecl decl;
     Token t;
 } {
     (
+        t = <GLOBAL>
+        global = Global()
```

Comment:
Do we need this keyword for globals? Feels a bit cleaner without

Reply:
Hm, I ran into a parsing error at first which was why I added the keyword, but I tried it again now and things seem to work. Removing.